extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -65,9 +84,25 @@ export class AmpSelector extends AMP.BaseElement {
       this.element.setAttribute('aria-disabled', 'true');
     }
 
+    const kbSupportMode = this.element.getAttribute('keyboard-support');
+    if (kbSupportMode == 'none') {
+      this.kbSupportMode_ = KB_SUPPORT.NONE;
+    } else if (kbSupportMode == 'select') {
+      this.kbSupportMode_ = KB_SUPPORT.SELECT;
+      user().assert(!this.isMultiple_,
+        '[keyboard-support=select] not supported for ' +
+        'multiple selection amp-selector');
+    } else {
+      this.kbSupportMode_ = KB_SUPPORT.FOCUS;
+    }
+    this.element.setAttribute('keyboard-support', this.kbSupportMode_);
```

Comment:
We normally don't set the "default" back to the attribute. (unless you need to target it in CSS)